prover/src/provers/split_prover.rs

```diff
@@ -0,0 +1,65 @@
+use super::Prover;
+use crate::contexts::SplitContext;
+use anyhow::Ok;
+use anyhow::Result;
+
+use elf::{endian::AnyEndian, ElfBytes};
+use num::ToPrimitive;
+use std::fs;
+
+use zkm::mips_emulator::state::{InstrumentedState, State, SEGMENT_STEPS};
+use zkm::mips_emulator::utils::get_block_path;
+
+#[derive(Default)]
+pub struct SplitProver {}
+
+impl SplitProver {
+    pub fn new() -> Self {
+        Self::default()
+    }
+}
+
+impl Prover<SplitContext> for SplitProver {
```

Comment:
SplitProver is not a prover, you can rename it to Executor, and it doesn't have to implement the Prover interface.

Reply:
done